specification/cognitiveservices/data-plane/Language/stable/2022-05-01/examples/SuccessfulAnalyzeTextJobsCancelRequest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "ApiVersion": "2022-03-01-preview",
```

Comment:
My scripts missed this one. I didn't expect this format of "ApiVersion". Will update my script, but please fix this version.

Reply:
updated

Reply:
updated